lib/cable_ready/sanity_checker.rb

```diff
@@ -9,40 +9,45 @@ class << self
     def check!
       return if CableReady.config.on_failed_sanity_checks == :ignore
       return if called_by_generate_config?
+      return if called_by_rake?
 
       instance = new
-      instance.check_javascript_package_version
+      instance.check_package_versions_match
       instance.check_new_version_available
     end
 
     private
 
     def called_by_generate_config?
-      ARGV.include? "cable_ready:initializer"
+      ARGV.include?("cable_ready:initializer")
+    end
+
+    def called_by_rake?
+      caller.find { |c| c.include?("/gems/rake-") }
```

Comment:
https://github.com/stimulusreflex/stimulus_reflex/pull/510#discussion_r671870456

Same comment as in the SR PR.